indico/modules/events/management/controllers/protection.py

```diff
@@ -147,3 +148,10 @@ class RHCategoryRolesJSON(RHManageEventBase):
     def _process(self):
         category_roles = CategoryRole.get_category_roles(self.event.category)
         return jsonify([serialize_category_role(cr, legacy=False) for cr in category_roles])
+
+
+class RHRegistrationFormsJSON(RHManageEventBase):  # TODO: Give this the API prefix instead of JSON?
+    PERMISSION = 'ANY'
```

Comment:
Materials can be managed by people who aren't managers as long as they have submission rights on something, so I think we need to give anyone who can access the event access to this endpoint :/

We could also check if the user has submission access on the event or any session/contribution but maybe that's overkill.

Reply:
I see what you mean, I also don't think it is critical to leave it open to the event. But I can always change it eventually.